src/YieldDistributor.sol

```diff
@@ -97,6 +100,7 @@ contract YieldDistributor is OwnableUpgradeable {
         minRequiredVotingPower = _minRequiredVotingPower;
         maxPoints = _maxPoints;
         cycleLength = _cycleLength;
+        yieldFixedSplit = _yieldFixedSplit;
```

Comment:
I don't like the name of this new variable. It overloads the term "split" to mean both "a precise amount of yield" and "the divisor used to calculate the split (previous definition)". I would change this to be `yieldFixedSplitDivisor` so it's more clear

Reply:
addressed with 5a940ad